vaadin-grid-flow-parent/vaadin-grid-flow/src/test/java/com/vaadin/flow/component/grid/GridSortingTest.java

```diff
@@ -293,6 +293,40 @@ public void setMultiSortWithPriorityNull_throws() {
         grid.setMultiSort(true, null);
     }
 
+    @Test
+    public void setMultiSortShiftClickOnly() {
+        grid.setMultiSort(true, true);
+
+        Assert.assertEquals("true",
+                grid.getElement().getProperty("multiSortOnShiftClick"));
```

Comment:
Prefer using `assertTrue` and `assertFalse` in the tests:

For example
```java
Assert.assertTrue(grid.getElement().getProperty("multiSortOnShiftClick", false));
```

Reply:
Done.